theme.css

```
@@ -1663,6 +1663,11 @@ ul.comparison-list > li em {
 }
 
 /** Filters */
+.filter-bar {
+  background-color: var(--s-background);
+  border-bottom: 1px solid var(--white-4);
```

Could you use white-1 for both of these?  That seems to match how the white theme looks just with the dark colors.

```suggestion
  background-color: var(--white-1);
  border-color: var(--white-1);
```